aten/src/ATen/templates/RegisterDispatchKey.cpp

```diff
@@ -37,10 +37,13 @@
 
 namespace at {
 
-namespace {
-
 ${dispatch_definitions}
```

Comment:
Do you plan on potentially generating functions with different namespaces in the same `Register{DispatchKey}.cpp` file? If it'll always be uniform per file, it might be cleaner to code-template the name of the namespace directly. So you can look something like `RegisterCPU.cpp` and easily say "oh yeah these functions are all in the same namespace"

Reply:
No, they'll either be anonymous, or in the cpu namespace (for example).